pkg/sdk/v1alpha1/kafkauser_types.go

```diff
@@ -85,3 +88,16 @@ func (spec *KafkaUserSpec) GetIfCertShouldBeCreated() bool {
 	}
 	return true
 }
+
+//GetAnnotations returns Annotations to use for Envoy generated Deployment and Pods
+func (spec *KafkaUserSpec) GetAnnotations() map[string]string {
+	return cloneAnnotationMap(spec.Annotations)
+}
+
+func cloneAnnotationMap(original map[string]string) map[string]string {
```

Comment:
Thanks, I cannot call util.MergeAnnotations here since it is in a different package. kafka-operator and this one is in the pkg/sdk. If I call that function I will introduce a concurrent dependency.

Reply:
but we could still have a single `cloneAnnotationMap` across the pkg/sdk package.